src/tests/ftest/server/daos_server_dump.py

```diff
@@ -25,6 +26,26 @@ def __init__(self, *args, **kwargs):
         self.start_servers_once = False
         self.setup_start_agents = False
 
+    def tearDown(self):
+        """Tear down after each test case."""
```

Comment:
The pylint warning is trying to tell you something, perhaps look into what the issue is rather than silence it.  It's normal to set a default value for all members in the __init__ function as well.

Reply:
Sure @ashleypittman , I was already aware of that, but I had presently no alternate solution and just wanted to silent it for now... But as @knard-intel has pointed me, the trick described at "[private-variables-and-methods-in-python](https://stackoverflow.com/questions/3385317/private-variables-and-methods-in-python)" may help me to solve original issue causing the pylint warning !!...